learn/how-tos/configuring-sap-hana-clouddb.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Connect To SAP Hana Cloud Database"
+id: ""
+---
+
+## Introduction
+
+The document explains the steps to connect to an SAP HANA CLOUD database from the WaveMaker application. SAP HANA is an in-memory, column-oriented, relational database management system offered by SAP. The platform provides a storage system  to store and retrieve data as requested by the applications. In addition, it performs advanced analytics.
```

Comment:
Could you also add additional statement saying that the user can only import the relation database schema using this document.

Reply:
Required changes were updated